ArduPlane/Parameters.cpp

```diff
@@ -88,23 +88,23 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(stab_pitch_down, "STAB_PITCH_DOWN",   2.0f),
 
-    // @Param: GLIDE_SLOPE_MIN
-    // @DisplayName: Glide slope minimum
-    // @Description: This controls the minimum altitude change for a waypoint before a glide slope will be used instead of an immediate altitude change. The default value is 15 meters, which helps to smooth out waypoint missions where small altitude changes happen near waypoints. If you don't want glide slopes to be used in missions then you can set this to zero, which will disable glide slope calculations. Otherwise you can set it to a minimum number of meters of altitude error to the destination waypoint before a glide slope will be used to change altitude.
+    // @Param: ALT_SLOPE_MIN
```

Comment:
maybe ALT_DELTA_MIN is better? The problem with "SLOPE" is it sounds like an angle

Reply:
Sure but if you read the description, the word "slope" appears multiple times as it describes what the functionality is - altitude slope calculations.

Reply:
Yes